collaboration-service/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:17-alpine
+
+WORKDIR /app
+
+# Copy package.json files into the container.
+COPY package*.json ./
+
+# Copy .next files into the container.
+COPY .next /app/.next
```

Comment:
iirc, .next files are regenerated at runtime, so it will be replaced with new ones whenever the app runs

Reply:
It has to be there. It's to copy the next folder, not generate.